src/test/java/com/kodilla/ecommercee/domain/OrderItemEntityTests.java

```diff
@@ -0,0 +1,130 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.repository.*;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import javax.transaction.Transactional;
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@Transactional
+@SpringBootTest
+public class OrderItemEntityTests {
+    @Autowired
+    OrderRepository orderRepository;
+
+    @Autowired
+    ProductRepository productRepository;
+
+    @Autowired
+    OrderItemRepository orderItemRepository;
+
+    @Autowired
+    GroupRepository groupRepository;
+
+    private OrderItem createOrderItem1() {
+        Order order1 = new Order();
+        Product product1 = new Product();
+        Group group1 = new Group();
```

Comment:
It seems to me that this object is not needed in this test because there is no direct relationship between OrderItem and Group.

Reply:
I thought so too, but Intellij returned the error about that, so I had to add it.

Reply:
I asked Andrzej about it and he confirmed that it's correct and apparently the relation between OrderItem and Group exists :)